eo-runtime/src/test/java/org/eolang/DataTest.java

```diff
@@ -72,11 +67,18 @@ void printsEmptyByteArray() {
     }
 
     @Test
-    @Disabled
     void printsString() {
-        MatcherAssert.assertThat(
-            new Data.ToPhi("Hello,\nдруг!").toString(),
-            Matchers.containsString("Hello,\\nдруг!")
+        final byte[] input = "Hello,\nдруг!".getBytes(StandardCharsets.UTF_8);
+        final StringBuilder output = new StringBuilder(0);
+        for (final byte elem : input) {
+            if (output.length() > 0) {
+                output.append('-');
+            }
+            output.append(String.format("%02X", elem));
+        }
+        MatcherAssert.assertThat(
```

Comment:
@c71n93 let's add assertion message to the test

Reply:
@maxonfjvipon sorry, I don't understand. what kind of message?

Reply:
@c71n93 check [this](https://github.com/objectionary/eo/blob/69471fcc350ae14e415eac139b5795ef9bdb0883/eo-maven-plugin/src/test/java/org/eolang/maven/ParseMojoTest.java#L107) as an example

Reply:
@maxonfjvipon done